src/cluster/purging.rst

```diff
@@ -93,6 +93,7 @@ Internal Replication
 ====================================
 Purge requests are replayed across all nodes in an eventually consistent manner.
 Internal replication of purges consists of two steps:
+
```

Comment:
If spacing doesn't render well, don't add blank lines here and on line 105. These will prevent RST/Sphinx from recognizing this as a numbered list.

Instead add the following text:

```rst
.. rst-class:: open
```

just after your section header on line 93. Check the output, but it should give you the effect you're  looking for.